core/ui/EditTemplate/fields.tid

```diff
@@ -126,16 +132,20 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 </div>
 </$reveal>
 </div>
+<$vars currentTiddler=<<newFieldValueTiddler>> currentField={{{ [<newFieldNameTiddler>get[text]] }}}>
```

Comment:
We should use $let instead of $vars.

Reply:
hmmm, From my point of view $let is only needed if you want to use a variable in a later parameter ... $vars is much less code to execute and imo much more lightweight and should be more performant

Reply:
> hmmm, From my point of view $let is only needed if you want to use a variable in a later parameter ... $vars is much less code to execute and imo much more lightweight and should be more performant

I don't believe there is any real performance difference and in fact we only have a new widget for backwards compatibility. $vars should be considered deprecated and slowly phased out from the core.

Thoughts @jermolene ?

Reply:
That's correct – the purpose of introducing the `<$let>` widget was to be able to deprecate the `<$vars>` widget.

It's not a great idea to only use `<$let>` when wanting to use a variable in a later parameter because it makes the complexity and cognitive load for users so much higher if they have to consider which to use each time. Much clearer and simpler to have a single, consistent recommendation.

Reply:
Should I add an additional commit replacing all occurrences of `<$vars>` in `fields.tid` with `<$let>` for consistency or should I only fix this one occurence?

Reply:
We need to do a systematic review at some point and replace $vars with $let, so perhaps restrict your changes to the new code.